integration-tests/package.json

```
@@ -15,7 +15,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@fluencelabs/deal-ts-clients": "file:../ts-client",
+    "@fluencelabs/deal-ts-clients": "file:fluencelabs-deal-ts-clients-0.7.0.tgz",
```

```suggestion
    "@fluencelabs/deal-ts-clients": "file:../ts-client",
```
Or else we need to build this archive manually on every change